### ISSUE TITLE
Accessibility - Select period is not announced to screen reader

### ISSUE THREAD

**Issue by chmnata** [CityofToronto/bdit_cot-vfh#57]
Issue

When a user makes selection to the Select Period, no announcement is sent to screen readers the data on the page have been updated.  Screen reader users need to know the content has changed.

Recommendation

Use a WAI-ARIA live region to inform screen reader users what item is currently shown.Locate the WAI-ARIA live region at the top of the page which is part of the COREJS framework.  

```<div class="js-aria-live sr-only" role="status" aria-live="polite"></div>```
Inject the announcement to the WAI-ARIA live region after a select is made in the 'Select Period' field.  "Showing top 10 PTC volume hotspots by neighbourhood, Post-PM peak (19:00–midnight)" 

```<div class="js-aria-live sr-only" role="status" aria-live="polite">```
Showing top 10 PTC volume hotspots by neighbourhood, Post-PM peak (19:00–midnight)</div>
